mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -887,26 +906,27 @@ static FuncOp createVerifyFuncOp(ModuleOp &module, OpBuilder &builder,
                                                   outputDimension[4]);
   verifyResultsBlock->push_back(indexOp4);
 
+  // clang-format off
   // scf.for %arg0 = %c0 to %c128 step %c1 {
   //  scf.for %arg1 = %c0 to %c30 step %c1 {
   //    scf.for %arg2 = %c0 to %c30 step %c1 {
   //      scf.for %arg3 = %c0 to %c128 step %c1 {
   //        scf.for %arg4 = %c0 to %c128 step %c1 {
-  //          %cpu_result = load %CpuResults[%arg0, %arg1, %arg2, %arg3, %arg4]
-  //          : memref<128x30x30x128x128xf32> %gpu_result = load
+  //          %cpu_result = load %CpuResults[%arg0, %arg1, %arg2, %arg3, %arg4] : memref<128x30x30x128x128xf32> %gpu_result = load
```

Comment:
comments here need re-layout.

Reply:
I intentionally kept this code in one line, because it looks more nature for me. But it's just me, I can change it back if that is the way you prefer.

Reply:
`%cpu_result` and `%gpu_result` shall be at different lines. moreover, the line for `%gpu_result` doesn't seem to be complete.